build/js/live-editor.tooltips.js

```diff
@@ -2134,6 +2134,7 @@ TooltipEngine.classes.colorPicker = TooltipBase.extend({
                 var $picker = $(this);
                 $picker.addClass("active");
                 down = true;
+                self.options.editor.scrubberActive = true;
```

Comment:
it might be better to use events to communicate changes in scrubber status to the outside world